docs/pages/blog/aggregation-functions.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Aggregate data like in Excel, but easier!
```

Comment:
I don't really like using words like "easy" and "simple" to describe things like this. Maybe we can approach this from a different angle.

```suggestion
title: Give users the power to intuitively aggregate data for more robust analysis
```

Reply:
I understand the argument about not using easy and simple in documentation, but this "easier" is not the same case I'd say